erdpy/docker.py

```diff
@@ -0,0 +1,57 @@
+import os
+import logging
+import subprocess
+from pathlib import Path
+from typing import List
+from erdpy.errors import DockerMissingError
+
+
+logger = logging.getLogger("build-with-docker")
+
+
+def is_docker_installed():
```

Comment:
Rename to `assert_docker_is_installed()`? Or make it to return a bool, instead of throwing an error, and throw it above, where called?

(usually, `is_...` functions should return boolean values).

Reply:
Changed `is_docker_installed` to return a boolean and moved `raise DockerMissingEror` on else statement.